src/app/features/tech-record/components/tyres-search/tyres-search.component.html

```diff
@@ -0,0 +1,112 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <dl class="govuk-summary-list">
+      <h1 class="govuk-heading-l">Tyre search</h1>
+      <h1 class="govuk-heading-m">Vehicle details</h1>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">Vehicle registration mark (VRM)</dt>
+        <dd id="current-vrm" class="govuk-summary-list__value">
+          <app-number-plate *ngIf="vehicleTechRecord?.vrms" [vrm]="currentVrm"></app-number-plate>
+        </dd>
+        <dd class="govuk-summary-list__value"></dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">Vehicle identification number (VIN)</dt>
+        <dd id="vin" class="govuk-summary-list__value">
+          {{ vehicleTechRecord?.vin | defaultNullOrEmpty }}
+        </dd>
+        <dd class="govuk-summary-list__value"></dd>
+      </div>
+    </dl>
+  </div>
+</div>
+
+<div [formGroup]="form" class="govuk-form-group">
+  <app-select
+    #searchFilter
+    label="Search filter"
+    [formControlName]="form.meta.children![1].name"
+    [name]="form.meta.children![1].name"
+    [options]="options!"
+  ></app-select>
+  <br />
+  <input
+    #searchTerm
+    [formControlName]="form.meta.children![0].name"
+    [name]="form.meta.children![0].name"
+    aria-describedby="term"
+    type="text"
+    class="govuk-input govuk-!-width-two-thirds"
+    (keyup.enter)="handleSearch(searchTerm.value, searchFilter.value)"
+  />
+  <button
+    class="govuk-button margin-right search-btn"
+    type="submit"
+    data-module="govuk-button"
+    enterkeyhint="search"
+    data-gem-track-click-module-started="true"
+    (click)="handleSearch(searchTerm.value, searchFilter.value)"
+  >
+    <svg
+      class="gem-c-search__icon"
+      width="25"
+      height="25"
+      viewBox="0 0 25 25"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      aria-hidden="true"
+      focusable="false"
+    >
+      <circle cx="12.0161" cy="11.0161" r="8.51613" stroke="currentColor" stroke-width="3"></circle>
+      <line x1="17.8668" y1="17.3587" x2="26.4475" y2="25.9393" stroke="currentColor" stroke-width="3"></line>
+    </svg>
+  </button>
+</div>
+
+<ng-container *ngIf="searchResults && !searchResults.length">
+  <h3 class="govuk-heading-m">0 search results</h3>
+  <p class="govuk-body govuk-!-font-size-19">No results found.</p>
+</ng-container>
+
+<ng-container *ngIf="searchResults && searchResults.length >= 1">
+  <h3 class="govuk-heading-m">{{ searchResults.length }} search result{{ searchResults.length > 1 ? 's' : '' }}</h3>
+</ng-container>
+
+<ng-container *ngIf="searchResults && searchResults.length">
+  <table class="govuk-table">
```

Comment:
```suggestion
<ng-container *ngIf="searchResults && searchResults.length">
  <h3 class="govuk-heading-m">{{ searchResults.length }} search result{{ searchResults.length > 1 ? 's' : '' }}</h3>
  <table class="govuk-table">
```
The `h3` can now be moved in the next container


OR:

Have one big wrapping container with `*ngIf="searchResults" and then a `*ngIf="searchResults.length; else noResults"`  on another container and an `ngTemplate` for the `noResults`